osmaxx/core/templates/osmaxx/login_logout_link.html

```diff
@@ -1,13 +1,19 @@
 {% load i18n staticfiles %}
-
+<li role="presentation" class="col-lg-push-1 col-md-push-1">
 {% if user.is_authenticated %}
-<a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
-    <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
-    {% trans 'Logout' %} <strong>{{ user.username }}</strong>
-</a>
+    <a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
+        <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
+        {% trans 'Logout' %}
+    </a>
+</li>
+<li role="presentation" class="col-lg-push-1 col-md-push-1">
+    <a href="{% url 'profile:edit_view' %}">
+        <strong>{{ user.username }}</strong>
+    </a>
 {% else %}
 <a href="{% url 'excerptexport:login' %}?next={% url 'excerptexport:index' %}">
     <img class="menu_icon" src='{% static "osmaxx/images/icon-login.png" %}' alt="Login" />
     {% trans 'Login' %}
 </a>
 {% endif %}
+</li>
```

Comment:
Whenever possible, please restrict the content of template control structures properly nested HTML. Otherwise it gets hard to see whether the rendering result will be well-formed in all cases.

Thus, either
```html
{% load i18n staticfiles %}
{% if user.is_authenticated %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
        <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
        {% trans 'Logout' %}
    </a>
</li>
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'profile:edit_view' %}">
        <strong>{{ user.username }}</strong>
    </a>
</li>
{% else %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'excerptexport:login' %}?next={% url 'excerptexport:index' %}">
        <img class="menu_icon" src='{% static "osmaxx/images/icon-login.png" %}' alt="Login" />
        {% trans 'Login' %}
    </a>
</li>
{% endif %}
```
or
```html
{% load i18n staticfiles %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
{% if user.is_authenticated %}
    <a href="{% url 'excerptexport:logout' %}?next={% url 'excerptexport:index' %}">
        <img class="menu_icon" src='{% static "osmaxx/images/icon-logout.png" %}' alt="Logout" />
        {% trans 'Logout' %}
    </a>
{% else %}
    <a href="{% url 'excerptexport:login' %}?next={% url 'excerptexport:index' %}">
        <img class="menu_icon" src='{% static "osmaxx/images/icon-login.png" %}' alt="Login" />
        {% trans 'Login' %}
    </a>
</li>
{% endif %}
{% if user.is_authenticated %}
<li role="presentation" class="col-lg-push-1 col-md-push-1">
    <a href="{% url 'profile:edit_view' %}">
        <strong>{{ user.username }}</strong>
    </a>
</li>
{% endif %}
```